src/index.ts

```diff
@@ -82,6 +82,14 @@ export interface ServerlessClamscanProps {
    * Set the performance mode of the EFS file system (Default: GENERAL_PURPOSE).
    */
   readonly efsPerformanceMode?: PerformanceMode;
+  /**
+   * Set the throughput mode of the EFS file system (Default: BURSTING), possible values: BURSTING, ELASTIC, PROVISIONED.
+   */
+  readonly efsThroughputMode?: ThroughputMode;
+  /**
+   * Set the Size for the throughputmode PROVISIONED, ignored otherwise
+   */
+  readonly efsProvisionedThroughputInMibps?: Size;
```

Comment:
```suggestion
  readonly efsProvisionedThroughputPerSecond: Size;
```

This property [was renamed](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_efs.FileSystem.html#provisionedthroughputpersecond)